### PR TITLE
PD: international opt-in additional work

### PR DESCRIPTION
**Pull request by breville**
Followup to https://github.com/code-dot-org/code-dot-org/pull/23235.

localhost-studio code org_3000_pd_international_workshop ipad pro

### CODE REVIEW

File: dashboard/test/controllers/api/v1/pd/international_opt_ins_controller_test.rb [hunk 1]
```diff
@@ -56,4 +56,31 @@ class Api::V1::Pd::InternationalOptInsControllerTest < ::ActionController::TestC
...
+
+    assert_does_not_create Pd::InternationalOptIn do
+      put :create, params: {
+        form_data: SAMPLE_FORM_DATA,
+        user: student
+      }
+      assert_response :forbidden
+    end
+
+    assert_response :forbidden
```
Comment (aoby) [inline dashboard/test/controllers/api/v1/pd/international_opt_ins_controller_test.rb]: this is redundant w line 69

File: dashboard/test/controllers/api/v1/pd/international_opt_ins_controller_test.rb [hunk 2]
```diff
@@ -56,4 +56,31 @@ class Api::V1::Pd::InternationalOptInsControllerTest < ::ActionController::TestC
...
+  test 'user required to create a new international opt-in' do
+    assert_does_not_create Pd::InternationalOptIn do
+      put :create, params: {
+        form_data: SAMPLE_FORM_DATA,
+        user: nil
+      }
+      assert_response :redirect
+    end
+
+    assert_response :redirect
```
Comment (aoby) [inline dashboard/test/controllers/api/v1/pd/international_opt_ins_controller_test.rb]: this too, with line 81

**Comment by breville** [comment]
This PR is now ready for review.

File: dashboard/config/locales/en.yml [hunk 3]
```diff
@@ -1041,7 +1041,7 @@ en:
       workshop_organizer: 'Workshop Organizer'
       workshop_facilitator: 'Workshop Facilitator'
       workshop_course: 'Workshop Course'
-      email_opt_in: 'I agree that Code.org can share my contact information and aggregate data about my classes with the Code.org International Partner in my country.'
+      email_opt_in: 'Can we email you about Code.org’s international program, updates to our courses, or other computer science news? (See our privacy policy)'
```
Comment (tanyaparker) [inline dashboard/config/locales/en.yml]: Is the privacy policy linked here? If we can't do links within this string, I'd prefer if `(See our privacy policy)` was a separate, linked string.

Comment (breville) [reply dashboard/config/locales/en.yml]: Good call, done.

**Pull request merged**